source/_components/owlet.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: page
+title: "Owlet"
+description: "Instructions on how to integrate Owlet baby monitor into Home Assistant."
+date: 2019-01-19
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: owlet.svg
+ha_category: Health
```

Comment:
@klaasnicolaas what's the standard now for categories? Is it enough with one category also for components that have platforms or should we always add the categories that correspond to the platform modules, eg binary_sensor etc?

Reply:
I think that platforms should be mentioned too.

Reply:
You will now mention both, the category and the platforms.

It is possible that if we have cleaned up a lot of nonsense pages this will still change, but then we can also better categorize and show everything to the users.